{{cookiecutter.project_name}}/.github/workflows/main.yml

```diff
@@ -53,6 +53,8 @@ jobs:
       - name: Upload coverage reports to Codecov with GitHub Action on Python 3.11
         uses: codecov/codecov-action@v4
         if: {% raw %}${{ matrix.python-version == '3.11' }}{% endraw %}
+        env:
```

Comment:
Reviewed the statement in https://github.com/fpgmaas/cookiecutter-uv/issues/28#issuecomment-2816747408. I have not yet tested with env, but I see no point, why it should work different from my approach in #28.